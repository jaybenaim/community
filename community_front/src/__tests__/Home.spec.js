import React from "react";
import { create } from "react-test-renderer";
describe("Home Component", () => {
  test("Shows at least one image li", () => {
    const li = create(<li></li>);
    expect(li.toJSON()).toMatchSnapshot();
  });
});
