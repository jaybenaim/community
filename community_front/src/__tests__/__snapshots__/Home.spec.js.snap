// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component Shows at least one image li 1`] = `<li />`;
